migration-engine/migration-engine-tests/tests/diagnose_migration_history/diagnose_migration_history_tests.rs

```diff
@@ -0,0 +1,431 @@
+use crate::*;
```

Comment:
only the last test is new, the rest was moved from `mod.rs` to this file with a more identifiable name